src/lib/reasoners/relation.ml

```diff
@@ -41,6 +41,9 @@ module Rel5 : Sig_rel.RELATION = Adt_rel
 
 module Rel6 : Sig_rel.RELATION = Ite_rel
 
+(* This source is unused. *)
+let src = Logs.Src.create ~doc:__MODULE__ ("AltErgoLib." ^ __MODULE__)
```

Comment:
Maybe we don't need it then? I think all the other sources are also unused, why single this one out?

Reply:
It is an artefact. I added them because I tried to add the source in the signature of `Sig_rel`, which was not a good idea. We do not want to build all the modules in `Parse_commands` just to setup up sources.

Reply:
In fact, it was not an artefact. I can remove `src` from `Sig_rel` but I have to introduce an extra signature in all the modules `*_rel.mli`.